tests/src/runtime/project.json

```diff
@@ -4,41 +4,72 @@
+    "System.Diagnostics.Process": "4.1.1-beta-24328-05",
     "System.Diagnostics.Tools": "4.0.2-beta-24328-05",
     "System.Diagnostics.Tracing": "4.1.1-beta-24328-05",
     "System.Dynamic.Runtime": "4.0.12-beta-24328-05",
     "System.Globalization": "4.0.12-beta-24328-05",
+    "System.Globalization.Calendars": "4.0.2-beta-24328-05",
     "System.IO": "4.1.1-beta-24328-05",
     "System.IO.FileSystem": "4.0.2-beta-24328-05",
     "System.IO.FileSystem.Primitives": "4.0.2-beta-24328-05",
     "System.Linq": "4.1.1-beta-24328-05",
+    "System.Linq.Queryable": "4.0.2-beta-24328-05",
+    "System.Linq.Expressions": "4.1.1-beta-24328-05",
     "System.Reflection": "4.1.1-beta-24328-05",
     "System.Reflection.Extensions": "4.0.2-beta-24328-05",
     "System.Resources.ResourceManager": "4.0.2-beta-24328-05",
     "System.Runtime": "4.1.1-beta-24328-05",
     "System.Runtime.Extensions": "4.1.1-beta-24328-05",
     "System.Runtime.Handles": "4.0.2-beta-24328-05",
     "System.Runtime.InteropServices": "4.2.0-beta-24328-05",
+    "System.Runtime.InteropServices.RuntimeInformation": "4.0.1-beta-24328-05",
+    "System.Runtime.Loader": "4.0.1-beta-24328-05",
+    "System.Runtime.Serialization.Json": "4.0.3-beta-24328-05",
+    "System.Runtime.Serialization.Primitives": "4.1.1",
+    "System.Runtime.Serialization.Xml": "4.1.2-beta-24328-05",
+    "System.Security.Cryptography.Algorithms": "4.2.1-beta-24328-05",
     "System.Text.Encoding": "4.0.12-beta-24328-05",
     "System.Text.Encoding.Extensions": "4.0.12-beta-24328-05",
     "System.Text.RegularExpressions": "4.2.0-beta-24328-05",
     "System.Threading": "4.0.12-beta-24328-05",
+    "System.Threading.AccessControl": "4.0.1-beta-24328-05",
     "System.Threading.Overlapped": "4.0.2-beta-24328-05",
     "System.Threading.Tasks": "4.0.12-beta-24328-05",
+    "System.Threading.Tasks.Parallel": "4.0.2-beta-24328-05",
     "System.Threading.ThreadPool": "4.0.11-beta-24328-05",
+    "System.Threading.Timer": "4.0.2-beta-24328-05",
     "System.Xml.ReaderWriter": "4.1.0-beta-24328-05",
     "System.Xml.XDocument": "4.0.12-beta-24328-05",
+    "System.Xml.XmlDocument": "4.0.2-beta-24328-05",
+    "System.Xml.XPath": "4.0.2-beta-24328-05",
+    "System.Xml.XPath.XmlDocument": "4.0.2-beta-24328-05",
+    "System.Numerics.Vectors": "4.1.2-beta-24328-05",
+    "Newtonsoft.Json": "7.0.1",
+    "Microsoft.CodeAnalysis.Compilers": "1.1.1",
+    "Microsoft.DotNet.xunit.performance": "1.0.0-alpha-build0035",
+    "Microsoft.DotNet.xunit.performance.analysis": "1.0.0-alpha-build0035",
+    "Microsoft.DotNet.xunit.performance.runner.Windows": "1.0.0-alpha-build0035",
+    "xunit": "2.1.0",
+    "xunit.console.netcore": "1.0.2-prerelease-00101",
+    "xunit.runner.utility": "2.1.0",
     "test_runtime": {
       "target": "project",
       "exclude": "compile"
     }
   },
   "frameworks": {
     "netcoreapp1.0": {
-      "imports": "dnxcore50"
+      "imports": [
+        "netcoreapp1.0",
```

Comment:
This self-referencing `netcoreapp1.0` TFM is extra, and can safely be removed from the imports list.